runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -85,22 +74,19 @@ public static boolean isMetricsReportingSetUp(MetricRegistry metrics) {
     /**
      * Start metrics reporting via the Dropwizard 'CsvReporter' file writer.
      * Reporting can be turned on and off via the properties file described
-     * in loadPropertiesFile()'s docs.  The report interval and report
+     * in loadVmProperties()'s docs.  The report interval and report
      * directory cannot be altered at runtime.
      *
      * @param metrics Metrics registry
      */
     public static void metricsReportingSetup(MetricRegistry metrics) {
         metrics.counter(mpTrigger);
```

Comment:
rename this, what is a mpTrigger ?

Reply:
mpTrigger was intended by the author for filtering out some of the metrics. The logic requires investigating & clean-up yet avoiding side effects. That's a task that I will do outside the scope of this enabling metrics PR.